examples/example-invoice-csv-export/src/main.php

```diff
@@ -0,0 +1,3 @@
+<?php
+
+declare(strict_types=1);
```

Comment:
this should probably log something about CLI not supported for this plugin

Reply:
I don't think so. With the new features many plugins won't have any use for CLI so logging that doesn't have any value. Instead I would consider making main.php optional.